client/src/components/relatedProducts/Modal.jsx

```diff
@@ -1,65 +1,63 @@
 import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import PropTypes from 'prop-types';
 
 // shows onClick star icon
 const Modal = (props) => {
-  const { comparedProduct } = props;
+  const { comparedProduct, currentProduct } = props;
+  const features = {};
+  if (comparedProduct.product !== undefined && comparedProduct.product.features !== undefined) {
+    // eslint-disable-next-line no-plusplus
+    for (let i = 0; i < currentProduct.features.length; i++) {
+      if (currentProduct.features[i].value === true) {
+        currentProduct.features[i].value = <FontAwesomeIcon icon={'faCheck'} />;
```

Comment:
lol have we added this icon to our icon library? looks like it's not showing up for me 👀

Reply:
check mark is only supposed to show if one of the values is the boolean true, which none of the products have that are currently rendering on the page. However, I will also add to the library in the "common" folder. I thought the "import fontawesome from @fortawesome..." as opposed to importing the file meant that adding to the library was not necessary. Sorry!

Reply:
added check mark to library